test/integration/ledger/node/tx.js

```diff
@@ -172,12 +172,36 @@ const transactions = {
+    hwSigningFiles: [signingFiles.multisigPayment0, signingFiles.multisigPayment1],
+    signedTxCborHex: '83a4008182582017ad99d6729537cf7e7bdaf5cc262c77bb0a578c2907a8948c7264d11cd58278000181825839000743d16cfe3c4fcc0c11c2403bbc10dbc7ecdd4477e053481a368e7a06e2ae44dff6770dc0f4ada3cf4cf2605008e27aecdb332ad349fda71a3b980b23021a0002bedd031a02407922a200828258200fa60d5831ee0be1f221ff93ba0a4fdeca5a5866e47569607b04e287fc9b5ec0584089f5706a7e0e7ac2d2f1f8d6631090b6bde479d887c331c27971734124aedb699cb194b430ff4929d669e7ea57d2855d20dfcf04901a8fbb17e5b90c3e18de018258202573f7cde2d182ebbcd63dc0940fd53064824e716a6d0238fa3780bac2933e2e5840f8a239f7c0f0587987aaf89496e23ad4574639458cf1ae3e158770a893e446d00f203cdfa74f2d63387df67bb1a18468490d8e37329f78d292599f68cfcff70001818201828200581c9a70dd7c77e9db9442b560a11446962e9d7c595274c587a62f8a0b618200581c9205690f6ea4b76742c6bdc5b79fb8a1727260089c71a64c7275320ef6',
+    network: 'TESTNET',
+  },
+  multisig_stakeAddressRegistrationCertificate: {
+    unsignedCborHex: '83a50081825820d5757f5ba07b09ee050692cebf3c605af12ff73c1eea4c99a769eae831850ec400018182583930de685e72586c4269087e282c9c7e78ba22082bce4a674977b4000e99b494d35f236093e7caed75d2b99b1e523cde935a6f4a2d276b9fb4011a3b6e88c7021a0002aa11031a0243037d048182008201581cb494d35f236093e7caed75d2b99b1e523cde935a6f4a2d276b9fb4019f8200581c9a70dd7c77e9db9442b560a11446962e9d7c595274c587a62f8a0b61fff6',
+    hwSigningFiles: [signingFiles.multisigPayment0],
+    signedTxCborHex: '83a50081825820d5757f5ba07b09ee050692cebf3c605af12ff73c1eea4c99a769eae831850ec400018182583930de685e72586c4269087e282c9c7e78ba22082bce4a674977b4000e99b494d35f236093e7caed75d2b99b1e523cde935a6f4a2d276b9fb4011a3b6e88c7021a0002aa11031a0243037d048182008201581cb494d35f236093e7caed75d2b99b1e523cde935a6f4a2d276b9fb401a200818258200fa60d5831ee0be1f221ff93ba0a4fdeca5a5866e47569607b04e287fc9b5ec0584037e4343aa3fbd6097b787ff67d3ecd391be7a8b558a6a0d1aff29fc5191a2b3ba618387f8a89ac4d41a61f7040594a09aa269f9c484cfba1c1bc8d2f239c330801818200581c9a70dd7c77e9db9442b560a11446962e9d7c595274c587a62f8a0b61f6',
+    network: 'TESTNET',
+  },
+  multisig_withdrawFromScriptStakeAddress: {
+    unsignedCborHex: '83a400818258202074b3783b7739037cabe8d6e9f2ca2ac8031c9d1a45334ae6853f7839709c5501018182583930de685e72586c4269087e282c9c7e78ba22082bce4a674977b4000e99b494d35f236093e7caed75d2b99b1e523cde935a6f4a2d276b9fb4011a3b6ba36a021a0002e55d05a1581df0b494d35f236093e7caed75d2b99b1e523cde935a6f4a2d276b9fb401009f8201828200581cf699c6400f85bdca54e44d0cad1f6141ce049a411c0d695fc30c3f738200581c4c139dd7e3a600fd1a4855b38cf8f731f39d3051791a5ede1f553d6cfff6',
+    hwSigningFiles: [signingFiles.multisigPayment0, signingFiles.multisigStake0, signingFiles.multisigStake1],
```

Comment:
Maybe it would be worthwhile to add a short comment describing why we need these signing files (or what is included in the tx which is not apparent from the name). It can be understood by analyzing the cbors, but that takes quite some time, and now you have it all readily available in your mind.

Applies to some of the other tests.

Reply:
Added the comments. Hopefully they are sufficient